### ISSUE TITLE
this project doesn't work anymore with Bazel 6

### ISSUE THREAD

**Issue by riccardodivirgilio** [fontanf/packingsolver#1]
Hi, I wanted to try this repo however this project (at least on my Mac machine) is spewing errors and closing.

the reason seems to be `:windows` constraint that is deprecated:

```
➜  packingsolver-0.7 bazel build -- //...
ERROR: /private/var/tmp/_bazel_rdv/3492d7386abad0d65c0c1bdd3f7f90df/external/bazel_tools/platforms/BUILD:89:6: in alias rule @bazel_tools//platforms:windows: Constraints from @bazel_tools//platforms have been removed. Please use constraints from @platforms repository embedded in Bazel, or preferably declare dependency on https://github.com/bazelbuild/platforms. See https://github.com/bazelbuild/bazel/issues/8622 for details.
ERROR: /private/var/tmp/_bazel_rdv/3492d7386abad0d65c0c1bdd3f7f90df/external/bazel_tools/platforms/BUILD:89:6: Analysis of target '@bazel_tools//platforms:windows' failed
ERROR: /private/var/tmp/_bazel_rdv/3492d7386abad0d65c0c1bdd3f7f90df/external/bazel_tools/platforms/BUILD:19:6: in alias rule @bazel_tools//platforms:x86_64: Constraints from @bazel_tools//platforms have been removed. Please use constraints from @platforms repository embedded in Bazel, or preferably declare dependency on https://github.com/bazelbuild/platforms. See https://github.com/bazelbuild/bazel/issues/8622 for details.
ERROR: /private/var/tmp/_bazel_rdv/3492d7386abad0d65c0c1bdd3f7f90df/external/bazel_tools/platforms/BUILD:19:6: Analysis of target '@bazel_tools//platforms:x86_64' failed
ERROR: /private/var/tmp/_bazel_rdv/3492d7386abad0d65c0c1bdd3f7f90df/external/boost/BUILD.bazel:1222:14: errors encountered resolving select() keys for @boost//:program_options
ERROR: Analysis of target '//packingsolver/rectangleguillotine:main' failed; build aborted: 
INFO: Elapsed time: 1.504s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
    Fetching repository @local_config_xcode; Fetching the default Xcode version
```

I removed all occurrences of :windows however it doesn't work because this is still used in external dependencies....

**Comment by fontanf** [comment]
Hi,

Can you try the `dev` branch? It should compile.

**Comment by riccardodivirgilio** [comment]
on the dev branch it works with both bezel  5.3.2 and 6.1.1 for my Mac machine.

**Comment by fontanf** [comment]
I fixed the `master` branch (f45f460)
So it should work as well if you prefer not using the `dev` branch
